cmd/boulder-wfe/main.go

```diff
@@ -41,9 +41,17 @@ type config struct {
 
 		TLS cmd.TLSConfig
 
-		RAService    *cmd.GRPCClientConfig
-		SAService    *cmd.GRPCClientConfig
-		NonceService *cmd.GRPCClientConfig
+		RAService *cmd.GRPCClientConfig
+		SAService *cmd.GRPCClientConfig
+		// GetNonceService contains a gRPC config for any nonce-service instances
+		// which we want to retrieve nonces from. In a multi-DC deployment this
+		// is should refer to any local nonce-service instances.
```

Comment:
typo: "this is should refer"
```suggestion
		// should refer to any local nonce-service instances.
```